.prettierrc

```
@@ -6,5 +6,7 @@
   "printWidth": 120,
   "jsxBracketSameLine": true,
   "jsxSingleQuote": true,
-  "endOfLine": "auto"
+  "endOfLine": "auto",
+  "importOrder": ["^([^(.|api\\-client|build\\-config)])", "api-client", "build-configs", "translations", "^./*"],
```

Don't you miss a `translations` in the first group? Also `^./*`does not match `../my-file` but it seems to work anyway right :thinking: 
```suggestion
  "importOrder": ["^([^(.|api\\-client|build\\-configs)])", "api-client", "build-configs", "translations", "^./*"],
```

---

You are right. I fixed it.